src/libraries/Microsoft.Extensions.Logging/tests/Microsoft.Extensions.Logging.Generators.Tests/Baselines/TestWithDynamicLogLevel.generated.txt

```diff
@@ -0,0 +1,62 @@
+// <auto-generated/>
+#nullable enable
+
+                    namespace Microsoft.Extensions.Logging.Generators.Tests.TestClasses
+                    {
+                        partial class TestWithDynamicLogLevel 
+                        {
+                            
+                            [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.Extensions.Logging.Generators", "6.0.0.0")]
+                            private readonly struct __M9Struct : global::System.Collections.Generic.IReadOnlyList<global::System.Collections.Generic.KeyValuePair<string, object?>>
```

Comment:
How come this one needs to generate a struct?

Reply:
As you see in the other generated file `TestWithOneParam.generated.txt`, LoggerMessage.Define takes LogLevel as argument. So if we were to alternatively support dynamic logging with Define APIs, we would need to have one Action per LogLevel and the generated method would call one of them based on the input LogLevel argument.

Reply:
ah, that makes sense. I thought it was just for `>6` args, but I guess there are other cases too. Thanks for the explanation.